sdp.c

```diff
@@ -22,69 +24,39 @@
 #include "debug.h"
 
 
-static su_home_t *home = NULL;
-
-
 /* SDP Initialization */
 int janus_sdp_init(void) {
-	home = su_home_new(sizeof(su_home_t));
```

Comment:
can these functions just go away completely and remove them from janus.c? If they are never going to fail now, there is not much point on having them, is there?